.gitignore

```diff
@@ -21,6 +21,8 @@ smart-gen-*
 static/id_buildfarm
 *prof
 keys/*.key
+/test-*.yaml
```

Comment:
What's this about?

Reply:
Aren't those always deleted by `cleanup` function?

Reply:
They are, but it still causes a turmoil in your git status while the test is running.. : -)

Reply:
Ideally then we'd use `mktemp -d` for intermediate artifacts and cleanup the directory after everything is done

Reply:
I agree, this could be the next improvement we make.